pkg/capi/edit/cluster.x-k8s.io.cluster.vue

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
```

Comment:
This is just a simple way for me to test the CCVariables component - I will remove it before merging.